rootfs/etc/nginx/lua/plugins.lua

```diff
@@ -5,8 +5,7 @@ local INFO = ngx.INFO
 local ERR = ngx.ERR
 
 local _M = {}
-local MAX_NUMBER_OF_PLUGINS = 10000
--- TODO: is this good for a dictionary?
+local MAX_NUMBER_OF_PLUGINS = 5
```

Comment:
I think this variable can be removed. The plugins are loaded once,i.e., we know how many we have.

Reply:
@agile6v what do you think of not preallocating `plugins` array? Since we load plugins when worker starts, I think it is OK to dynamically expand the array.

--

There are more optimization we can do here by keying plugins by NGINX phase to avoid iterating through all plugins in all supported phase. But that can be done in a separate PR.

Reply:
yes. I'm in favor of removing the preallocated array.  
Changed.